webapp/app/js/model/jobListModel.js

```diff
@@ -25,15 +25,15 @@ KylinApp.service('JobList',function(JobService,$q){
     this.jobs={};
     this.jobFilter = {
         cubeName : null,
-        timeFilterId : 1,
+        timeFilterId : 0,
```

Comment:
Hi, it's more reasonable to keep the default setting as "last one week".

Reply:
Hi, i hava kept the default setting as "last one week" and make it possible to get the job time filter default value from kylin configuration file, thank you very much!